core/src/commonMain/kotlin/entity/User.kt

```diff
@@ -37,6 +37,10 @@ public open class User(
     override val id: Snowflake
         get() = data.id
 
+    /**
+     * Returns true if the user is the same as the bot.
+     */
+public val isSelf: Boolean get() = id == kord.selfId
```

Comment:
There is some funny indentation going on here
```suggestion
   public val isSelf: Boolean get() = id == kord.selfId
   
```

Reply:
mb, only checked the code on github web

Reply:
still looks weird tho